TOLERANCE: 0.2
MIN_CLUSTER_SIZE: 5
MAX_CLUSTER_SIZE: 10000
# windowsのペイントでHだけ180/255をかける
LOWER_RED_H: 0
LOWER_RED_S: 100
LOWER_RED_V: 80
UPPER_RED_H: 10
UPPER_RED_S: 200
UPPER_RED_V: 200

LOWER_BLUE_H: 10
LOWER_BLUE_S: 150
LOWER_BLUE_V: 170
UPPER_BLUE_H: 30
UPPER_BLUE_S: 260
UPPER_BLUE_V: 220

LOWER_YELLOW_H: 22
LOWER_YELLOW_S: 40
LOWER_YELLOW_V: 100
UPPER_YELLOW_H: 30
UPPER_YELLOW_S: 120
UPPER_YELLOW_V: 200

LOWER_GREEN_H: 56 # 60
LOWER_GREEN_S: 150 # 100
LOWER_GREEN_V: 50 # 95
UPPER_GREEN_H: 77 # 80
UPPER_GREEN_S: 220 # 180
UPPER_GREEN_V: 100 # 190

LOWER_WHITE_H: 5
LOWER_WHITE_S: 5
LOWER_WHITE_V: 245
UPPER_WHITE_H: 25
UPPER_WHITE_S: 25
UPPER_WHITE_V: 255